pkg/generators/forwarding/fluentd/output_conf_kafka_test.go

```diff
@@ -81,8 +81,8 @@ var _ = Describe("Generating external kafka server output store config block", f
            brokers broker1-kafka.svc.messaging.cluster.local:9092
            default_topic topic
            ssl_ca_cert '/var/run/ocp-collector/secrets/some-secret/ca-bundle.crt'
-           ssl_client_cert '/var/run/ocp-collector/secrets/some-secret/tls.crt'
-           ssl_client_cert_key '/var/run/ocp-collector/secrets/some-secret/tls.key'
+           ssl_client_cert "#{File.exist?('/var/run/ocp-collector/secrets/some-secret/tls.crt') ? '/var/run/ocp-collector/secrets/some-secret/tls.crt' : use_nil}"
```

Comment:
Don't like repeating the filename, opportunity for future error. Suggest wraping this expression in a func existing_filename() or similar. Like the rubyites say, keep your code DRY :)

Reply:
It's already the case in `templates.go` with `{{ .SecretPath }}`. You look here on the expected output of a test case ;)

Reply:
Doh!
/lgtm